src/components/AddItem.js

```diff
@@ -40,6 +69,7 @@ const AddItem = () => {
           type="text"
           value={groceryItem}
           onChange={onGroceryItemInputChange}
+          required
```

Comment:
I love that you took into consideration the edge case of a user not inputting anything and submitting the form by adding "required" to all of the inputs.